res/values/themes.xml

```diff
@@ -202,5 +202,23 @@
     <item name="colorSubLabel">@android:color/system_neutral1_300</item>
     <item name="emoji_button_bg">?colorKeyActivated</item>
     <item name="emoji_color">@android:color/system_accent1_900</item>
+  </style>
+	<style name="PastelDarkBlue" parent="@style/BaseTheme">
+    <item name="android:isLightTheme">false</item>
+    <item name="colorKeyboard">#333334</item>
+    <item name="colorKey">#11111B</item>
+    <item name="colorKeyActivated">#22222A</item>
+    <item name="colorLabel">#CDD6F4</item>
+    <item name="colorLabelActivated">#ffffff</item>
+    <item name="colorLabelLocked">#8AADF4</item>
+    <item name="colorSubLabel">#DEE5E3</item>
+    <item name="keyBorderWidth">1.0dip</item>
+    <item name="keyBorderWidthActivated">1.0dip</item>
+    <item name="keyBorderColorLeft">#8AADF4</item>
```

Comment:
I think this theme could be a great candidate for the "monet" color system that uses the colors from the system theme.

Here's an experimentation:
```xml
	<style name="PastelDarkBlue" parent="@style/BaseTheme">
    <item name="android:isLightTheme">false</item>
    <item name="colorKeyboard">#333334</item>
    <item name="colorKey">#11111B</item>
    <item name="colorKeyActivated">#22222A</item>
    <item name="colorLabel">@android:color/system_accent1_300</item>
    <item name="colorLabelActivated">#ffffff</item>
    <item name="colorLabelLocked">#8AADF4</item>
    <item name="colorSubLabel">@android:color/system_accent1_500</item>
    <item name="keyBorderWidth">1.0dip</item>
    <item name="keyBorderWidthActivated">1.0dip</item>
    <item name="keyBorderColorLeft">@android:color/system_accent1_400</item>
    <item name="keyBorderColorTop">@android:color/system_accent1_400</item>
    <item name="keyBorderColorRight">@android:color/system_accent1_400</item>
    <item name="keyBorderColorBottom">@android:color/system_accent1_400</item>
    <item name="emoji_button_bg">?colorKeyActivated</item>
    <item name="emoji_color">#000000</item>
  </style>
```
![shot](https://github.com/user-attachments/assets/b91c6d9b-e6a7-4362-a7fa-a36d416d8b3b)

Reply:
As far as I can see, "monet" is a style that is themed according to the colors of the material you? , well, actually if it was born from there is the theme but it was actually to combine with the discord

For me it's fine (the current "monet" reminded me when you put random colors in the CMD with [Color AF] :v)

Reply:
I would have to try my black friend's phone ( It's autoracist so I don't know if it counts as self-love or that the CPU celeron did him wrong) that has "android 13" (my hell range [it gives to cook on the screen] is android 10 XD)

Reply:
Thanks for trying the monet thing but I think we should drop this idea.

Reply:
I'll leave it on hold, I'm not going to abandon it completely, (what I did abandon is the accessibility, I don't understand it) in what I have free (after finishing some soundtracks (and some sprites because we took the designers out because they were lazy XD) I dedicate myself to trying the Monet (material you) and the custom colors

a, and maybe to put a sound for the keys 


(uff I came up with a tremendous idea, bad apple on the keyboard :0)